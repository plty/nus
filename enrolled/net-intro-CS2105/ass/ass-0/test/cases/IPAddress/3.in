11000000101010001110101000010001
