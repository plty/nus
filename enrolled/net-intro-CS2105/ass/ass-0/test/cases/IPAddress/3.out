192.168.234.17
